VARIABLES.md

```diff
@@ -406,20 +414,28 @@ Default:  "{{kafka_broker_configure_additional_brokers}}"
 
 ### kafka_broker_user
 
-Only use to customize Linux User Kafka Service runs with. User must exist on host.
+Only use to customize Linux User Kafka Service runs with. Default user is cp-kafka
```

Comment:
Modify this variable to customize the Linux User that the Kafka Service runs with. Default user is cp-kafka.